packages/theme/src/cli/commands/theme/update/run.ts

```diff
@@ -0,0 +1,16 @@
+import ThemeCommand from '../../../utilities/theme-command.js'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+
+export default class UpdateInit extends ThemeCommand {
+  static description = `Run an 'update_extension.json' script in a theme.`
+
+  static hidden = true
+
+  static flags = {
+    ...globalFlags,
+  }
+
+  async run(): Promise<void> {
+    // empty.
```

Comment:
I'm going to introduce this command in a separate PR to keep this one from getting too long.